### PR TITLE
add table components based on Vue2.x

### PR DESCRIPTION
**Pull request by huangshuwei**
A powerful table components based on Vue2.x

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -303,7 +303,8 @@
   - [Vue 2048 (Vue + Webpack)](https://pengfu.github.io/vue-2048/) by [@pengfu](https://github.com/pengfu): Popular 2048-Game implemented using Vue,Webpack,Sass,ES6
   - [Vue Simple PWA](https://github.com/BosNaufal/vue-simple-pwa) by [BosNaufal](https://github.com/BosNaufal)
   - [Tour of Heroes (Vue 2.0)](https://github.com/aweber1/tour-of-heroes-vue2): A Vue 2.0 port of the Angular 2.0 [Tour of Heroes](https://angular.io/docs/ts/latest/tutorial/) demo app. Highlights: ES6/7, render functions, JSX, [revue](https://github.com/revue/revue) (redux bindings for Vue), [vue-router](https://github.com/vuejs/vue-router), [Airbnb eslint](https://github.com/airbnb/javascript/tree/master/packages/eslint-config-airbnb-base), webpack. by [@aweber1](https://github.com/aweber1)
-  - [vue-table-pagination](https://github.com/echoVic/vue-table-pagination) A table with pagination by [echovic](https://github.com/echovic)
+  - [
```
Comment (kazupon) [inline README.md]: breaking format ...

Comment (huangshuwei) [reply README.md]: I have changed the document into English:
https://github.com/huangshuwei/vue-easytable

**Comment by huangshuwei** [comment]
@kazupon 
Thanks for patience.
I have changed the document into English:
https://github.com/huangshuwei/vue-easytable

File: README.md (hunk 1 above)
Comment (kazupon) [reply README.md]: Thanks.
I checked your repo.
It' a okay.

However, `README.md` is breaking ...
Could you fix this it?

**Comment by huangshuwei** [comment]
I do not know where the problem is, can I repeat a PR?

**Comment by huangshuwei** [comment]
you can close this PR

**Pull request closed without merging**